CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Added
 
-- `RPMBuilder::source_date()` method for clamping modification time of files. This allows reproducible builds when included files are re-generated (having the same content but with different mtimes).
+- `RPMBuilder::source_date_epoch()` method for clamping modification time of files and build time of the package. This allows reproducible builds when included files are re-generated (having the same content but with different mtimes).
 
```

Comment:
Change this to `Breaking Change` and explain that `build_time` as a solitary option is going away

Reply:
I've marked removal of `build_time` as a breaking change and kept `source_date_epoch` as an addition.